obspy/taup/tau.py

```diff
@@ -246,6 +246,7 @@ def plot_times(self, phase_list=None, plot_all=True, legend=False,
 
     def plot_rays(self, phase_list=None, plot_type="spherical",
                   plot_all=True, legend=False, label_arrivals=False,
+                  indicate_wave_type=False,
                   show=True, fig=None, ax=None):
```

Comment:
Oh i wanted to mention that earlier.. can you please move the new kwarg to the end of the list? I know it logically fits better here, but this kinda isn't backwards compatible (in the highly unlikely case that somebody uses this with 6+ args ;)